package.json

```diff
@@ -26,5 +26,6 @@
     "invalid date",
     "invalid format",
     "invalid date format"
-  ]
-}
\ No newline at end of file
+  ],
+  "dependencies": {}
```

Comment:
Do we need this? Currently, we are not dependent on any other packages right?

Reply:
No need...not sure where it came from